h/views/api_flags.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from pyramid import security
+from pyramid.httpexceptions import HTTPNoContent
+
+from memex.views import api_config
+
+
+@api_config(route_name='api.flag_annotation',
+            request_method='POST',
+            link_name='api.flag_annotation',
```

Comment:
Ahhh, sorry, [answered my own question](https://github.com/hypothesis/h/blob/9a0a77f92b4f17253ce3b659a59460bb1db8355d/src/memex/views.py#L69-L85). I missed this change.